.ruby-version

```diff
@@ -1 +1 @@
-2.6.6
+2.7.5
```

Comment:
I *think* this file just affects `rbenv`.  It's not referenced in the gemspec or anything else.

So we should change it, so that `rbenv` gets a newer version of Ruby, but it's not something users of this gem need to care or know about (eg, you wouldn't add "we renamed the test task in our github action workflow" to a CHANGELOG).